pennylane/ops/functions/equal.py

```diff
@@ -206,7 +206,7 @@ def _equal_operators(
             if qml.math.get_interface(params_1) != qml.math.get_interface(params_2):
                 return False
 
-    return getattr(op1, "inverse", False) == getattr(op2, "inverse", False)
+    return True
```

Comment:
What cases does this line now cover?

Reply:
If none of the other cases have return False, then we know they are equal.

We could potentially rearrange the order of checks to end with something like:
```
return op1.hyperparameters == op2.hyperparameters
```
but I don't think it's that important of a change.